### PR TITLE
Add volume test

### PR DESCRIPTION
**Pull request by zbud-msft**
**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: src/sonic-eventd/tools/rsyslog_volume_test.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    handlers = [
+        logging.FileHandler("debug.log"),
+        logging.StreamHandler(sys.stdout)
+    ]
+)
+
+def read_events_from_file(count):
+    logging.info("Reading from file generated by events_tool")
+    lines = 0
+    with open('STDOUT', 'r') as file:
```
Comment (renukamanavalan) [inline src/sonic-eventd/tools/rsyslog_volume_test.py]: The STDOUT being used as file name is tool's bug, which I should fix. So if user would not give any file, I should send it to STDOUT, not file "STDOUT"

Can you please explicitly pass a file name ?
If not this  code will be broken, when I fix the events_tool.

File: src/sonic-eventd/tools/rsyslog_volume_test.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    return proc
+
+def run_test(file, count):
+    # log messages to see if events have been received
+    tool_proc = start_tool()
+
+    time.sleep(2) # buffer for events_tool to startup
+
+    log_messages_cnt = 0
+    logging.info("Generating logger messages\n")
```
Comment (renukamanavalan) [inline src/sonic-eventd/tools/rsyslog_volume_test.py]: Can we have a default behavior with no file?  -- like sending few hardcoded events 10 or 15
Also can we default count to a higher value, like 1000

File: src/sonic-eventd/tools/rsyslog_volume_test.py (hunk 1 above)
Comment (zbud-msft) [reply src/sonic-eventd/tools/rsyslog_volume_test.py]: Done

File: src/sonic-eventd/tools/rsyslog_volume_test.py (hunk 2 above)
Comment (zbud-msft) [reply src/sonic-eventd/tools/rsyslog_volume_test.py]: Done

File: src/sonic-eventd/tools/rsyslog_volume_test.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+            for _ in range(count):
+                subprocess.run(command, shell=True, stdout=subprocess.PIPE)
+
+    time.sleep(2) # some buffer for all events to be published to file
+    read_events_from_file(count * log_messages_cnt)
+    tool_proc.terminate()
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-f", "--file", nargs='?', const='', default='', help="File containing log messages to be generated and read by rsyslog_plugin")
```
Comment (renukamanavalan) [inline src/sonic-eventd/tools/rsyslog_volume_test.py]: If you set required = True, the python parser will fail on no arg.
Need not change. 
More a tip.

File: src/sonic-eventd/tools/events_volume_test.py [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        command = "logger -p local0.notice -t {}".format(line)
+        subprocess.run(command, shell=True, stdout=subprocess.PIPE)
+
+    time.sleep(2) # some buffer for all events to be published to file
+    read_events_from_file(file, count)
+    tool_proc.terminate()
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-p", "--process", nargs='?', const ='', default='', help="Process that is spitting out log")
```
Comment (renukamanavalan) [inline src/sonic-eventd/tools/events_volume_test.py:53]: By setting required=true, you can avoid empty arg check later.

**Review by renukamanavalan — APPROVED** [review]

**Pull request merged**